src/Walkthrough.js

```diff
@@ -22,23 +22,7 @@ class WalkthroughComponent extends React.Component {
         <UserBubble>{t("walkthrough.these_are_my_sins")}</UserBubble>
         <PriestBubble text={t("walkthrough.your_confessor_may_offer")} />
         <PriestBubble text={t("walkthrough.your_confessor_will_assign")} />
-        <UserBubble>
-          <Trans t={t} i18nKey="walkthrough.act_of_contrition">
-            <p>My God, I am sorry for my sins with all my heart.</p>
-            <p>
-              In choosing to do wrong and failing to do good, I have sinned
-              against You, whom I should love above all things.
-            </p>
-            <p>
-              I firmly intend with Your help to do penance, to sin no more, and
-              to avoid whatever leads me to sin.
-            </p>
-            <p>
-              Jesus Christ suffered and died for us. In His name, my God, have
-              mercy.
-            </p>
-          </Trans>
-        </UserBubble>
+        <UserBubble>{t("prayers.act_of_contrition")}</UserBubble>
```

Comment:
> I hope you find commit https://github.com/kas-catholic/confessit-web/commit/4053d02c43eb646733288f6074a5107a38aac197 acceptable, I figure it's useless to translate the exact same text twice in two different contexts.

Seems fine I guess, I don't have strong feelings either way.

----

But this is the wrong translation, it should be `act_of_contrition_text` to get the complete prayer, not just the title.

![image](https://github.com/kas-catholic/confessit-web/assets/78571006/0225eddd-0696-4104-b117-8b53a42fc32e)


https://deploy-preview-67--confessit-web.netlify.app/

Reply:
dang didn't even double check that! I just switched to `act_of_contrition_text`, to find another problem: when using `Trans` components, HTML is preserved. But when using a `t()` function to retrieve a translation, html is not preserved. Only way to keep it, is to use a wrapper `<div>` setting the `dangerouslySetInnerHtml` property with the text of the translation and the indication to not escape html (`interpolation: { escapeValue: false }`).

Does this still work, or is it starting to get messy?

Reply:
Well I guess there is no such thing as an empty self-closing `Trans` component, it was causing tests to fail. I had seen it as an example on [stackoverflow](https://stackoverflow.com/a/72177021/394921), but I guess it isn't quite correct. So I'm just using an exact copy of the `Trans` component from `Prayers.js`, this seems to work fine and tests pass...